src/components/Questionnaire/structured/handlers.ts

```diff
@@ -195,10 +204,51 @@ export const structuredHandlers: {
       }));
     },
   },
+  consent: {
+    getRequests: async (consents, { patientId }) =>
+      consents.map((consent) => ({
+        url: `/api/v1/patient/${patientId}/consent/`,
+        method: "POST",
+        body: {
+          ...consent,
+          file: undefined as unknown as File,
+        },
+        reference_id: "consent",
+      })),
+    afterRequest: async (data, res) => {
```

Comment:
i would name it `onSuccess` or `postSubmit`.

`afterRequest` doesn't sound correct. it is actually after the response of the request right? 🤔